### PR TITLE
RavenDB-22362 Unify audit log lines

### PR DESCRIPTION
**Pull request by arekpalinski**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-22362/Unify-audit-log-lines

### Additional description

Continuation of:

- https://github.com/ravendb/ravendb/pull/18526
- https://github.com/ravendb/ravendb/pull/18605 - applied PR comments

### Type of change

- [ ] Bug fix
- [ ] Regression bug fix
- [x] Optimization
- [ ] New feature

### How risky is the change?

- [x] Low 
- [ ] Moderate 
- [ ] High
- [ ] Not relevant

### Backward compatibility

- [ ] Non breaking change
- [ ] Ensured. Please explain how has it been implemented?
- [ ] Breaking change
- [x] Not relevant

### Is it platform specific issue?

- [ ] Yes. Please list the affected platforms.
- [x] No

### Documentation update

- [ ] This change requires a documentation update. Please mark the issue on YouTrack using `Documentation Required` tag.
- [x] No documentation update is needed 

### Testing by Contributor

- [ ] Tests have been added that prove the fix is effective or that the feature works
- [ ] Internal classes added to the test class (e.g. entity or index definition classes) have the lowest possible access modifier (preferable `private`) 
- [ ] It has been verified by manual testing

### Testing by RavenDB QA team

- [ ] This change requires a special QA testing due to possible performance or resources usage implications (CPU, memory, IO). Please mark the issue on YouTrack using `QA Required` tag.
- [ ] No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- [ ] Yes. Please list the affected features/subsystems and provide appropriate explanation
- [x] No

### UI work

- [ ] It requires further work in the Studio. Please mark the issue on YouTrack using `Studio Required` tag.
- [x] No UI work is needed

### CODE REVIEW

File: src/Raven.Server/Documents/Handlers/Admin/AdminIndexHandler.cs [hunk 1]
```diff
@@ -81,7 +81,7 @@ private async Task PutInternal(bool validatedAsAdmin)
                     
                     if (LoggingSource.AuditLog.IsInfoEnabled)
                     {
-                        LogAuditFor(Database.Name, $"Index {indexDefinition.Name} PUT with definition: {indexToAdd}");
+                        LogAuditFor(Database.Name, "PUT", $"Index {indexDefinition.Name} with definition: {indexToAdd}");
```
Comment (gregolsky) [inline src/Raven.Server/Documents/Handlers/Admin/AdminIndexHandler.cs]: For analyzer we user '' and here we don't

Comment (arekpalinski) [reply src/Raven.Server/Documents/Handlers/Admin/AdminIndexHandler.cs]: done

File: src/Raven.Server/Integrations/PostgreSQL/Handlers/PostgreSqlIntegrationHandler.cs [hunk 2]
```diff
@@ -201,7 +201,7 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
 
                     if (LoggingSource.AuditLog.IsInfoEnabled)
                     {
-                        LogAuditFor(Database.Name, $"Postgres integration. User {userToDelete.Username} DELETE");
+                        LogAuditFor(Database.Name, "DELETE", $"User '{userToDelete.Username}' Postgres integration");
```
Comment (gregolsky) [inline src/Raven.Server/Integrations/PostgreSQL/Handlers/PostgreSqlIntegrationHandler.cs]:
```suggestion
                        LogAuditFor(Database.Name, "DELETE", $"User '{userToDelete.Username}' in Postgres integration");
```

File: src/Raven.Server/Web/Authentication/TwoFactorAuthenticationHandler.cs [hunk 3]
```diff
@@ -94,7 +94,7 @@ public async Task ValidateTotp()
             
             if (LoggingSource.AuditLog.IsInfoEnabled)
             {
-                LogAuditFor(nameof(TwoFactorAuthenticationHandler), $"successfully authenticated with two factor auth for {period} (until: {DateTime.UtcNow.Add(period)}). Has limits: {hasLimits}");
+                LogAuditFor(nameof(TwoFactorAuthenticationHandler), "AUTH", $"Successfully with 2FA for {period} (until: {DateTime.UtcNow.Add(period)}). Has limits: {hasLimits}");
```
Comment (gregolsky) [inline src/Raven.Server/Web/Authentication/TwoFactorAuthenticationHandler.cs]:
```suggestion
                LogAuditFor(nameof(TwoFactorAuthenticationHandler), "AUTH", $"2FA session open for {period} (until: {DateTime.UtcNow.Add(period)}). Has limits: {hasLimits}");
```

File: src/Raven.Server/Web/System/DatabasesHandler.cs [hunk 4]
```diff
@@ -116,7 +116,7 @@ public async Task ModifyTopology()
 
                 if (LoggingSource.AuditLog.IsInfoEnabled)
                 {
-                    LogAuditFor("DbMgmt", $"Database '{dbName}' topology is being modified " +
+                    LogAuditFor("DbMgmt", "MODIFY", $"Database '{dbName}' topology. " +
```
Comment (gregolsky) [inline src/Raven.Server/Web/System/DatabasesHandler.cs]: We usually say in the studio that "topology changed". Maybe CHANGE?

**Review by gregolsky — APPROVED** [review]

**Pull request merged**